.env

```diff
@@ -39,6 +39,10 @@ BASIC_AUTH_PASSWORD=password
 BASIC_AUTH_PROXY_TIMEOUT=600
 
 ### DATABASE ###
+# Airbyte Bootloader environment variables
+BOOTLOADER_MIGRATION_BASELINE_VERSION=0.29.0.001
+RUN_DATABASE_MIGRATION_ON_STARTUP=true
```

Comment:
Any cases where we do not want to run migrations?

Reply:
@gosusnp Don't know.  This env var exists for some reason, so I suspect it was added for a particular case.  We can obviously remove this in the future if we deem that it is not needed.